flink-python/pyflink/datastream/connectors/cassandra.py

```diff
@@ -0,0 +1,1648 @@
+################################################################################
+#  Licensed to the Apache Software Foundation (ASF) under one
+#  or more contributor license agreements.  See the NOTICE file
+#  distributed with this work for additional information
+#  regarding copyright ownership.  The ASF licenses this file
+#  to you under the Apache License, Version 2.0 (the
+#  "License"); you may not use this file except in compliance
+#  with the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+from enum import Enum
+from typing import List, Union
+
+from pyflink.common import Duration, Configuration
+from pyflink.java_gateway import get_gateway
+from pyflink.util.java_utils import to_jarray
+
+# ---- Classes introduced to construct the MapperOptions ----
+
+
+class ConsistencyLevel(Enum):
+    """
+    The consistency level
+    """
+    ANY = 0
+    ONE = 1
+    TWO = 2
+    THREE = 3
+    QUORUM = 4
+    ALL = 5
+    LOCAL_QUORUM = 6
+    EACH_QUORUM = 7
+    SERIAL = 8
+    LOCAL_SERIAL = 9
+    LOCAL_ONE = 10
+
+    def _to_j_consistency_level(self):
+        JConsistencyLevel = get_gateway().jvm.com.datastax.driver.core.ConsistencyLevel
+        return getattr(JConsistencyLevel, self.name)
+
+
+# ---- Classes introduced to construct the ClusterBuilder ----
+
+
+class ProtocolVersion(Enum):
+    """
+    Versions of the native protocol supported by the driver.
+    """
+    V1 = 0
```

Comment:
```suggestion
    V1 = 0   # MiniCassandraVersion 1.2.0
```

Add some comments as above to give users a basic understanding of which version should be used?

Reply:
This is the part about ClusterBuilder, I will create a new PR and fix it for easy review~